app/lib/shared/versions.dart

```diff
@@ -6,6 +6,13 @@ import 'package:pub_semver/pub_semver.dart';
 
 import 'utils.dart' show isNewer;
 
+final String runtimeVersion = '2018.3.8';
```

Comment:
Maybe add a comment here, to update this whenever one of the other versions change?